static/styles/compose.css

```diff
@@ -759,6 +759,7 @@ a.compose_control_button.hide {
     #compose-textarea,
     #preview_message_area {
         max-height: none !important;
+        height: 0;
```

Comment:
Let's add a comment explaining what CSS this is overriding. On the face of it, setting the height to 0 seems like it might be a bug.

Reply:
I remember needing to set the height to 0 when dealing with an element with data-simplebar and with flex-grow: 1 [here](https://github.com/zulip/zulip/pull/22370/files#diff-c7b3b66d8e1de59be1c99412b775677b90ac3cf9812f527eba8848738cac5e9fR3340), to let it be scrollable. Here the case seems to be the same. If I remember correctly, it's a quirk of simplebar, that when applied to a non list element, it needs it's height to be set to 0